README.md

```diff
@@ -110,6 +110,53 @@ Based on the discussion in [#79](https://github.com/js-reporters/js-reporters/is
 
 Additional properties (not defined here) can be added to the Assertion object.
 
+## Details
+
+For details please check out the [docs](https://github.com/js-reporters/js-reporters/blob/add-examples/docs) and especially the [example](https://github.com/js-reporters/js-reporters/blob/add-examples/docs/example.md) which presents a testsuite and its reporting data based on the standard presented above.
+
+For implementations examples please check [Usage of the adapters](#usage-of-the-adapters) and [Integrations](#integrations).
+
+## Usage of the adapters
+
+Listen to the events and receive the emitted data:
+
+```js
+var runner;
+
+// Attach the adapter.
+if (QUnit) {
+  runner = new JsReporters.QUnitAdapter(QUnit);
```

Comment:
I wonder if we should add something like `var runner = JsReporters.guess()`, which looks at globals like this if/else/else does and throws an error when it can't find any framework.